jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/awsrules/cloudtrail/EnableCloudTrailCloudwatchLogsIntegration.java

```diff
@@ -0,0 +1,63 @@
+package com.tmobile.cloud.awsrules.cloudtrail;
+import com.nimbusds.oauth2.sdk.util.MapUtils;
+import com.tmobile.cloud.awsrules.utils.PacmanUtils;
+import com.tmobile.cloud.constants.PacmanRuleConstants;
+import com.tmobile.pacman.commons.PacmanSdkConstants;
+import com.tmobile.pacman.commons.exception.InvalidInputException;
+import com.tmobile.pacman.commons.policy.Annotation;
+import com.tmobile.pacman.commons.policy.BasePolicy;
+import com.tmobile.pacman.commons.policy.PacmanPolicy;
+import com.tmobile.pacman.commons.policy.PolicyResult;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+import java.util.*;
+
+@PacmanPolicy(key = "enable-cloudtrail-cloudwatch-logs-integration", desc = "This rule checks that CloudTrail trails are integrated with CloudWatch Logs", severity = "critical", category = PacmanSdkConstants.SECURITY)
+public class EnableCloudTrailCloudwatchLogsIntegration extends BasePolicy {
+
+    private static final Logger logger = LoggerFactory.getLogger(EnableCloudTrailCloudwatchLogsIntegration.class);
+
+    @Override
+    public PolicyResult execute(final Map<String, String> ruleParam, Map<String, String> resourceAttributes) {
+
+        logger.debug("========EnableCloudTrailCloudwatchLogsIntegration started=========");
```

Comment:
1. Please do not use repeating characters in logs. We don't have any standard approach for them and it makes a mess in logs for others.
2. If you log that something has started you're supposed to log that smth is stopped as well. Don't see it here.

Reply:
done

Reply:
don't see the change

Reply:
The log is removed